test/unit/pipeline/test_create_forecast.py

```diff
@@ -125,17 +125,16 @@ def test_create_forecast_pipeline_happy_flow_2_days(self):
     def test_create_forecast_pipeline_happy_flow_4_days(self):
         """Test the happy flow of the forecast pipeline with a trained model."""
         # Load prediction job and forecast data
-        pj = TestData.get_prediction_job(pid=307)
-        pj["model_type_group"] = "default"
+        pj, modelspecs = TestData.get_prediction_job(pid=307)
         forecast_data = TestData.load("reference_sets/307-test-data.csv")
         col_name = forecast_data.columns[0]
         forecast_data.loc["2020-11-26 00:00:00":"2020-12-01", col_name] = None
-        pj["feature_names"] = forecast_data.columns[1:]
+        modelspecs.feature_names = forecast_data.columns[1:]
```

Comment:
We can set the feature names after loading the modelspecs with load_model() if it's still required after loading the model